lib/events/azsessions/azsessions.go

```diff
@@ -86,7 +86,7 @@ func partPrefix(upload events.StreamUpload) string {
 }
 
 // partName returns the name of the blob for a specific part in an upload.
-func partName(upload events.StreamUpload, partNumber int64) string {
+func partName(upload events.StreamUpload, partNumber int32) string {
```

Comment:
Out of curiousity, why did we change part number to an int32?

Reply:
Because `s3.UploadPartInput.PartNumber` in `aws-sdk-go-v2` is now `int32`, it was `int64` in v1

Reply:
Right, but `azsessions` is Azure code. It seems strange that we have to change _all_ of our cloud storage backends in order to update an AWS dependency that is only used in one of them.

I wonder if this PR would be simpler if we left the interface unchanged and did the conversion in the `s3sessions` code.